dcrutil/appdata.go

```diff
@@ -31,12 +31,8 @@ func appDataDir(goos, appName string, roaming bool) string {
 
 	// Get the OS specific home directory via the Go standard lib.
 	homeDir, err := os.UserHomeDir()
-
-	// Fall back to standard HOME environment variable that works
-	// for most POSIX OSes if the directory from the Go standard
-	// lib failed.
-	if err != nil || homeDir == "" {
-		homeDir = os.Getenv("HOME")
+	if err != nil {
+		homeDir = ""
```

Comment:
I don't guess it hurts to do this, but the `UserHomeDir` method documentation does explicitly document:

> // If the expected variable is not set in the environment, UserHomeDir
> // returns either a platform-specific default value or a non-nil error.

I confirmed that by looking at the implementation as well.

Given that, this really could just be:

```Go
	// Get the OS specific home directory via the Go standard lib.
	//
	// Note that the error is intentionally ignored here since an empty string
	// for the home directory will be returned in the case of error which is the
	// desired behavior.
	homeDir, _ := os.UserHomeDir()
```